test/integration/ListingItemObjectData.test.ts

```diff
@@ -0,0 +1,163 @@
+import { Types, Core, Targets } from '../../src/constants';
+import { TestUtil } from './lib/TestUtil';
+import { TestDataService } from '../../src/api/services/TestDataService';
+
+import { ValidationException } from '../../src/api/exceptions/ValidationException';
+import { NotFoundException } from '../../src/api/exceptions/NotFoundException';
+
+import { ListingItemObjectData } from '../../src/api/models/ListingItemObjectData';
+import { ListingItemTemplate } from '../../src/api/models/ListingItemTemplate';
+
+import { ListingItemObjectType } from '../../src/api/enums/ListingItemObjectType';
+
+import { ListingItemObjectDataService } from '../../src/api/services/ListingItemObjectDataService';
+import { ProfileService } from '../../src/api/services/ProfileService';
+import { ListingItemTemplateService } from '../../src/api/services/ListingItemTemplateService';
+
+import { ListingItemObjectDataCreateRequest } from '../../src/api/requests/ListingItemObjectDataCreateRequest';
+import { ListingItemObjectDataUpdateRequest } from '../../src/api/requests/ListingItemObjectDataUpdateRequest';
+import { TestDataCreateRequest } from '../../src/api/requests/TestDataCreateRequest';
+
+describe('ListingItemObjectData', () => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = process.env.JASMINE_TIMEOUT;
+    const log: LoggerType = new LoggerType(__filename);
+    const testUtil = new TestUtil();
+
+    let testDataService: TestDataService;
+    let listingItemObjectDataService: ListingItemObjectDataService;
+    let profileService: ProfileService;
+    let listingItemTemplateService: ListingItemTemplateService;
+
+    let createdId;
+    let createdListingItemObject;
+
+    const testData = {
+        key: 'Screensize',
+        value: '17.8 inch',
+        listing_item_object_id: null
+    } as ListingItemObjectDataCreateRequest;
+
+    const testDataUpdated = {
+        key: 'gpu',
+        value: 'NVidia 950',
+        listing_item_object_id: null
+    } as ListingItemObjectDataUpdateRequest;
+
+    beforeAll(async () => {
+        await testUtil.bootstrapAppContainer(app);  // bootstrap the app
+
+        testDataService = app.IoC.getNamed<TestDataService>(Types.Service, Targets.Service.TestDataService);
+        listingItemObjectDataService = app.IoC.getNamed<ListingItemObjectDataService>(Types.Service, Targets.Service.ListingItemObjectDataService);
+        profileService = app.IoC.getNamed<ProfileService>(Types.Service, Targets.Service.ProfileService);
+        listingItemTemplateService = app.IoC.getNamed<ListingItemTemplateService>(Types.Service, Targets.Service.ListingItemTemplateService);
+
+        // clean up the db, first removes all data and then seeds the db with default data
+        await testDataService.clean();
+
+        const defaultProfile = await profileService.getDefault();
+        const createdListingItemTemplate = await testDataService.create<ListingItemTemplate>({
+            model: 'listingitemtemplate',
```

Comment:
modify the existing listingitemtemplatecreaterequest.json's from testdata to contain listingitemobjects and use those in your tests. that way the same data will be used in other tests too and in the future we will have better data to run the tests with.